### ISSUE TITLE
show changes interactively

### ISSUE THREAD

**Issue by phmarek** [gregsexton/gitv#13]
When doing "git rebase -i" it would be nice to have a vertical split, with one side showing the rebase buffer and the other one the files changed and the diff related to the change the cursor is on.

When moving the cursor in the rebase buffer, the other pane should be updated immediately, without any intervention.

The same could be used for (fugitive's) Gstatus - showing the file diff in the other pane, or for Gitv browse mode - don't require an Enter.

Thank you very much for this very nice plugin!

**Comment by rbong** [comment]
Merging has been added since this issue was posted. Interactive rebasing is next on my list of new features.

**Comment by rbong** [comment]
Interactive rebasing is now possible. More automated rebasing might be possible in the future, like squashing ranges.

**Comment by rbong** [comment]
While the idea for having a rebase window that knows about the preview window has not been incorporated, you can now mark commits for rebase instructions before starting the rebase and view the rebase instructions next to the commit messages. I think an inline solution is nicer and still solves the problem.

I never addressed the other request. Unfortunately, updating the preview window while moving around will break some of the bindings introduced since this issue has been opened.
